website/content/docs/agent/options.mdx

```diff
@@ -137,11 +146,38 @@ The options below are all specified on the command-line.
   capture, it is possible to use [`discard_check_output`](#discard_check_output).
 
 - `-client` ((#\_client)) - The address to which Consul will bind client
-  interfaces, including the HTTP and DNS servers. By default, this is "127.0.0.1",
+  interfaces, including the HTTP, HTTPS, gRPC and DNS servers. By default, this is "127.0.0.1",
   allowing only loopback connections. In Consul 1.0 and later this can be set to
   a space-separated list of addresses to bind to, or a [go-sockaddr]
   template that can potentially resolve to multiple addresses.
 
+  <CodeBlockConfig hideClipboard>
+  
+  ```shell
+  # Bind consul client interfaces to private IPv4 interfaces.
+  $ consul agent -dev -client '{{ GetPrivateInterfaces | exclude "type" "ipv6" | join "address" " " }}'
+  ```
+  
+  </CodeBlockConfig>
+
+  <CodeBlockConfig hideClipboard>
+
+  ```shell
+  # Bind consul client interfaces to private IP addresses and loopback
+  $ consul agent -dev -client '{{ GetPrivateInterfaces | join "address" " " }} {{ GetAllInterfaces | include "flags" "loopback" | join "address" " " }}'
```

Comment:
Could it make sense to show what the output list/array of values would be when calling multiple interfaces? Could serve as a nice safety check for a reader if they don't know what the command string looks like, and have to run `ps -ef` on a server somewhere. Just a thought!

Reply:
Do you mean show how this would be interpreted by `go-sockaddr` or what this example looks like on the command line if I ran `ps -ef`?

The parsing will happen inside of Consul. If I ran the above example and then used `ps -ef` to see the command line it would output the text from the example.

Reply:
The computed value of the `consul agent` command string. So for example, something like `consul agent-dev -client lo en0 en1`  etc etc. 

But based on your comment the string template is present when running `ps -ef`, and rendered inside of consul. If that's the case then my suggestion doesn't make sense, and you can disregard.